File: app/src/main/java/com/alliejc/wcstinder/trackmyswing/APIInterface.java
package com.alliejc.wcstinder.trackmyswing;

import com.alliejc.wcstinder.trackmyswing.Dancer;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

/**
 * Created by acaldwell on 3/18/18.
 */

public interface APIInterface {

    @GET("{division}/{role}")
    Call<List<Dancer>> getDancers(@Path("division") String division, @Path("role") String role);
}

File: app/src/main/java/com/alliejc/wcstinder/trackmyswing/APIInterface.kt
package com.alliejc.wcstinder.trackmyswing

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path
import retrofit2.http.Query

/**
 * Created by acaldwell on 4/8/18.
 */
interface APIInterface {
    @GET("{division}/{role}")
    fun getDancers(@Path("division")division:String, @Path("role")role:String): Call<MutableList<Dancer>>


}